CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Make MSRV of 1.87.0 explicit.
 
 - Implement `Default` for `CapacityError`
+- Implement `defmt::Format` for `CapacityError`
```

Comment:
Could you please add the missing `.` here and on the previous line (I know you didn't add that but let's fix it) while we're at it.